### PR TITLE
Telephone_Number

### PR DESCRIPTION
**Pull request by Marwa-Sadat**
1. *Objective:* An app was developed to verify if a given telephone number meets specified formatting rules, ensuring accurate and consistent phone number entries. 📞✔
2. *Inputs:* Users can input their telephone numbers into a designated field. ✏
3. *Validation Logic:* The app accurately checks the formatting of the entered telephone numbers. 🔄
4. *Error Handling:* Users are alerted if their input does not match the required format. 🚫❗
5. *User Interface:* A clean and intuitive interface was designed to enhance user experience. 🎨📱
6. *Responsiveness:* The app is fully responsive and functions well on all devices. 🌐📲
7. *Deployment:* The app was successfully hosted online, ensuring easy access for users. 🌐
Screenshot 2024-06-29 122216

### CODE REVIEW

**Comment by ahzamir** [comment]
@Marwa-Sadat 

- I can not see your partner feedback

**Comment by Marwa-Sadat** [comment]
Screenshot 2024-07-02 155945
Hi @zainabbehzad  I hope you are well, please check my project .

**Comment by Marwa-Sadat** [comment]
Screenshot 2024-07-04 220309
Hi @zinabbehzad I hope you are well, please check my project.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,89 @@
...
+2. **Input a String**:
+  Enter the string you want to check when prompted.
+
+3. **View the Result**:
+The program will output whether the given string is a Telephone_Number or not.
+
+## Demo
+
+![Demo](https://github.com/Marwa-Sadat/Telephone_Number/assets/168111110/7dcaedf8-877b-468b-8c13-984e9ee09802)
+
```
Comment (zainabbehzad) [inline README.md:50]: please add Demo of this project.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,89 @@
...
+### Prerequisites
+
+- [Your Programming Language] installed on your machine.
+
+### Steps
+
+1. **Clone the Repository**:
+  bash
+  git clone https://github.com/Marwa-Sadat/Telephone_Number.git
+  cd palindrome-checker
```
Comment (zainabbehzad) [inline README.md]: Please add name of this project

**Review by zainabbehzad — CHANGES_REQUESTED** [review]
@Marwa-Sadat 

🎉 Great Design Implemented
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Telephone Number Validation project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your design is responsive
🧩Your repo has a professional README.md file
* Please do not forget to add Deployment Link of your project in README.md
In your rREADME.md file you need some changes:

**Comment by Marwa-Sadat** [comment]
@zainabbehzad I hope you are well, I did the feedback, please check my project again.

**Review by zainabbehzad — APPROVED** [review]
@Marwa-Sadat 

🎉 Great Design Implemented
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Telephone Number Validation project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your design is responsive
✔Your repo has a professional README.md file
* Please do not forget to add Deployment Link of your project in README.md

**Comment by Marwa-Sadat** [comment]
@ahzamir Hi sir  I hope you are well ,please check my project  sir.

**Comment by Marwa-Sadat** [comment]
@ahzamir Hi sir I hope you are well ,please check my project sir.

**Review by ahzamir — APPROVED** [review]
### Hi @Marwa-Sadat 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- Well done [OPTIONAL] just add some colors to your page to make it look more cool
- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Telephone Number Validator project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**